README.md

```diff
@@ -3,24 +3,38 @@
+
+## Membership to the 'BrownUniversity' organization within GitHub
+
+Once you have an account, you will be able to create your own repositories and work with other public ones on the site. To participate in Brown-related projects, you will need to associate your GitHub account with one or more Brown organizations. The main organization for Brown is [BrownUniversity](https://github.com/BrownUniversity), and you can join it by following [this link](https://github.com/orgs/BrownUniversity/sso/sign_up). Other organizations (CIS, CCV, Libraries, etc.) may have other methods to join.
 
-GitHub's accounts are globally unique to GitHub. Your **Brown ShortID** may **not** be available on GitHub. This is **OK**. Since you will be authenticating via Brown's Shibboleth what name you have on GitHub won't really matter. Your Shibboleth login authenticates you.
+## Accessing Single Sign On protected resources within GitHub from the command line (or other tools that don't have a Shibboleth interface)
 
-You may already have a GitHub account for personal or non-Brown related code work. You may use this for Brown's GitHub access as well. Brown does **not** have any special access to other projects on GitHub. 
+GitHub provides a [desktop client](https://desktop.github.com/) for macOS and Windows, but most developers prefer to use the command line or other clients. Many IDEs and [developer-oriented text editors](https://atom.io/) can make use of GitHub integration. To integrate your desktop experience with GitHub outside of the official apps, there are additional steps required to create a 'key' that you can authorize connections with.
+
+Please follow the steps in [this document](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) to generate a private key (this must NOT be shared or stored in a location where it is at risk of being compromised) and authorize it for use with GitHub. This procedure creates a trust relationship between applications within your desktop environment, Shibboleth, and GitHub.
+
+You can use the same SSH key pair for other services besides GitHub, but that is out-of-scope for this document. To read more on the subject, please start [here](https://www.ssh.com/ssh/public-key-authentication).
 
 ## Learn Git and GitHub
 
 If you are not already familiar with git and GitHub, Brown has access to several excellent online courses at [LinkedIn Learning](https://www.linkedin.com/learning/).
 
-Here are 2 to get you started:
+Here are three helpful resources to get you started:
 
 * [Learning GitHub](https://www.linkedin.com/learning/learning-github/version-control-and-collaboration-with-github) - 2hrs 11min: Includes bare basics of using git.  
 * [GitHub Essential Training](https://www.linkedin.com/learning/github-essential-training) - 2hrs 48min: Covers CI concepts and more advanced git usage.
+* [Git Cheet Cheet from GitLab](https://about.gitlab.com/images/press/git-cheat-sheet.pdf)
```

Comment:
Is this spelling correct for cheat sheet?

Reply:
Whoops. Fixed that. Good eyes!